controllers/fio_controller.go

```diff
@@ -82,6 +82,21 @@ func (r *FioReconciler) newJob(ctx context.Context, cr *perfv1alpha1.Fio, crRef
 
 	backoffLimit := int32(0)
 
+	imagePullPolicy := corev1.PullPolicy(cr.Spec.Image.PullPolicy)
+	if imagePullPolicy == "" {
```

Comment:
I think this is also taken care of by Kubernetes. 
My hunch is that we can pass in as-is (including nil value).

Reply:
nil (more precisely: empty) value might be handled somehow, but our sophisticated approach implemented in the if, definitely not